src/main/scala/org/broadinstitute/dsde/workbench/sam/api/StandardSamUserDirectives.scala

```diff
@@ -28,13 +28,19 @@ trait StandardSamUserDirectives extends SamUserDirectives with LazyLogging with
 
   def withActiveUser(samRequestContext: SamRequestContext): Directive1[SamUser] = requireOidcHeaders.flatMap { oidcHeaders =>
     onSuccess {
-      getActiveSamUser(oidcHeaders, directoryDAO, tosService, samRequestContext).unsafeToFuture()
+      getActiveSamUser(oidcHeaders, directoryDAO, tosService, samRequestContext).unsafeToFuture().map(samUser => {
+        logger.info(s"Handling request for active Sam User: $samUser")
```

Comment:
Do we need all the info from SamUser, or just the userId?

Reply:
I think it is better to log the full user for debugging azure/google or whatever